.rubocop.yml

```diff
@@ -1,6 +1,6 @@
 AllCops:
   NewCops: enable
-  TargetRubyVersion: 2.7
+  TargetRubyVersion: 3.0
```

Comment:
I haven't done any analysis of the Ruby ecosystem so I trust that supporting >=3.0 is fine.